Added where downloaded certificates go (#1098)
OctopusDeploy/docs: docs/deployment-examples/certificates/export-certificate.md

position: 60
---

> Certificates can be downloaded from the Octopus Portal to the machine you are accessing the portal from.  The certificate may be exported in any of the [supported file-formats](/docs/deployment-examples/certificates/file-formats.md), or exactly as it was originally uploaded.

Author: ```suggestion
Certificates can be downloaded from the Octopus Portal to your local machine.  The certificate may be exported in any of the [supported file-formats](/docs/deployment-examples/certificates/file-formats.md), or exactly as it was originally uploaded.
```
```suggestion
Certificates can be downloaded from the Octopus Portal to the machine you are accessing the portal from.  The certificate may be exported in any of the [supported file-formats](/docs/deployment-examples/certificates/file-formats.md), or exactly as it was originally uploaded.
```
